packages/react/src/runtime/is-node.tsx

```diff
@@ -11,6 +11,5 @@
  * ```
  */
 export const isNodeEnvironment = (): boolean => {
```

Comment:
Did you have any thoughts on this vs. renaming `isBrowserEnvironment`? More change I suppose?

Reply:
I think it makes sense to keep the interface the same unless we have a major reason to refactor.